src/Documents.ts

```diff
@@ -817,6 +821,8 @@ class Documents implements Pointer.Library, Pointer.Validator, ObjectSchema.Reso
 		// TODO: Decorate additional behavior (app, etc.). See also updatePersistedDocument() method
 		// TODO: Make it dynamic. See also updatePersistedDocument() method
 		if( LDP.Container.Factory.hasRDFClass( document ) ) LDP.PersistedContainer.Factory.decorate( document );
```

Comment:
PersistedContainer "extends" PersistedRDFSource so `PersistedContainer.Factory.decorate` should call `PersistedRDFSource.Factory.decorate`